src/commands/job/execute.ts

```diff
@@ -102,11 +102,28 @@ export async function execute(
       macroVars?.macroVars
     )
     .catch(async (err) => {
+      if (err?.name === 'NoSessionStatus') {
```

Comment:
I think an (err instanceof NoSessionStatus) check would be nicer here.

Reply:
done